src/Sylius/Bundle/ApiBundle/Doctrine/QueryItemExtension/OrderMethodsItemExtension.php

```diff
@@ -49,7 +50,17 @@ public function applyToItem(
             return;
         }
 
-        if ($operationName !== Request::METHOD_DELETE) {
+        if (
+            $operationName === Request::METHOD_GET ||
+            $operationName === ApiShopRequestTypes::SHOP_GET ||
+            $operationName === ApiShopRequestTypes::ADJUSTMENTS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::ITEMS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::ITEMS_ADJUSTMENTS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::SHIPMENTS_METHODS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::PAYMENTS_METHODS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::SHIPMENTS_GET_SUBRESOURCE ||
+            $operationName === ApiShopRequestTypes::PAYMENTS_GET_SUBRESOURCE
+        ) {
```

Comment:
This logic should be at least extracted to a separate method because right now it is duplicated in two places (just with different comparison signs). If I see correctly, this if is not tested at all while it brings 11 additional paths of execution. 

Using `in_array()` could clean additionally a code a little bit, but it will just hide complexity. Tests are needed anyway. Maybe it would be a good idea to check `get` with `strpos`?